preprocess_angha_dataset.py

```diff
@@ -1,36 +1,53 @@
-"""Main script for preprocessing the anghadataset"""
+#!/usr/bin/env python3
+"""Main script for preprocessing the anghadataset. Must be run from the root dir of the project"""
+import sys
 from pathlib import Path
 
-from decompile.preprocessing import preprocess
+from decompile.preprocessing.preprocess import preprocess
+from decompile.preprocessing.preprocess import collect_c_files
+from decompile.preprocessing.preprocess import create_jsonl_and_standardize
```

Comment:
I understand the motivation behind such imports style (I believe it is to motivate developers to reduce the dependencies per module and consequently increasing code modularity). However, this is a script not a module file, so it's ok to merge all of these imports into one import 

```python
from decompile.preprocessing.preprocess import preprocess, collect_c_files, create_jsonl_and_standardize
```

Reply:
Sounds reasonable. I also followed this convention to reduce merge conflicts in the future and to clarify which functions to use as an interface. I think there is a better way to import using `__all__` but I am not sure.
Changed it for now.